site/source/docs/tools_reference/emcc.rst

```diff
@@ -78,6 +78,8 @@ Options that are modified or new in *emcc* are listed below:
 ``-Oz``
 	Like ``-Os``, but reduces code size even further. This can effect both bitcode generation and JavaScript.
 
+	For JavaScript, this turns on some code size reduction optimizations that can take a significant amount of compilation time.
```

Comment:
Can we combine `-Oz -s EVAL_CTORS=0` to get the previous `-Oz`? Not sure if we generally need it, but it might be useful for someone to cross-compare new and old `-Oz`?